helpers/pginit.js

```diff
@@ -0,0 +1,120 @@
+import pg from 'pg';
+import config from '../config.js';
+
+module.exports=function(){
+
+const conString = `postgres://${config.userName}:${config.password}@${config.dbServerLocation}:${config.serverPort}/${config.dbName}`;
+const client = new pg.Client(conString);
+
+client.connect();
+client.query(`CREATE TABLE IF NOT EXISTS users(
+    id              serial        PRIMARY KEY,
+    fullname        varchar(64)   UNIQUE        NOT NULL,
+    userrole        varchar(20)   NOT NULL,
+    passhesh        varchar(40)   NOT NULL,
+    salt            varchar(16)   NOT NULL,
+    email           varchar(50)   NOT NULL,
+    is_premium      boolean       NOT NULL,
+    phone_number    varchar(24)   NOT NULL,
+    is_deleted      boolean       NOT NULL,
+    avatar          varchar(100),
+    aboutMe         text
+);`); 
+
+client.query(`CREATE TABLE IF NOT EXISTS recipes_category(
+    id            serial        PRIMARY KEY,
+    name        varchar(100)    UNIQUE        NOT NULL,
+    is_deleted    boolean       NOT NULL
+)`);
+client.query(`CREATE TABLE IF NOT EXISTS recipes(
+    id            serial        PRIMARY KEY,
+    title         varchar(100)  UNIQUE        NOT NULL,
+    description   text          NOT NULL,
+    is_deleted    boolean       NOT NULL,
+    owner_id      integer       references users(id),
+    category_id   integer       references recipes_category(id),
+    photo         varchar(100),
+    rating        integer
+)`);
+client.query(`CREATE TABLE IF NOT EXISTS ingradients(
+    id            serial        PRIMARY KEY,
+    name          varchar(30)   UNIQUE      NOT NULL,
+    is_deleted    boolean       NOT NULL,
+    photo         varchar(100)   
+)`);
+client.query(`CREATE TABLE IF NOT EXISTS calc_card(
+    id              serial    PRIMARY KEY,
+    resipe_id       serial  references recipes(id),
+    ingredient_id   serial  references ingradients(id)
+)`);
+client.query(`CREATE TABLE IF NOT EXISTS ingradient_map(
+    id              serial          PRIMARY KEY,
+    ingredient_id   integer         references ingradients(id), 
+    is_deleted      boolean         NOT NULL,
+    lat             real,
+    lon             real,
+    price           varchar(10)
+)`);
+client.query(`CREATE TABLE IF NOT EXISTS tags(
+    id           serial         PRIMARY KEY,
+    name         varchar(30)    UNIQUE      NOT NULL
```

Comment:
@python4eg What do you mean? Should I rename `name` like `tag description` or add a new field?